genmod/score_variants/compound_scorer.py

```diff
@@ -83,109 +135,134 @@ def run(self):
             # This is a dictionary on the form {'variant_id: rank_score}
             rank_scores = {}
             # We are now going to score the compounds
-            for variant_id in variant_batch:
-                # First we store the scores for each variant in a dictionary
-                variant = variant_batch[variant_id]
-                rank_score_entry = variant['info_dict'].get('RankScore', '')
-                
-                # We need to loop through the families
-                # This entry looks like <family_id>:<rank_score>, <family_id>:<rank_score>
-                for family_rank_score in rank_score_entry.split(','):
-                    family_rank_score = family_rank_score.split(':')
-                    
-                    #TODO check if correct family id
-                    # Right now we assume that there is only one family in the vcf
-                    family_id = family_rank_score[0]
-                    rank_score = int(family_rank_score[-1])
-                    
-                    rank_scores[variant_id] = rank_score
+            for rank_score_type in RANK_SCORE_TYPE_NAMES:
+                # Prepare rank_scores dict to contain legacy RankScore and RankScoreNormalized compound scores
+                rank_scores.update({rank_score_type: dict()})
+
+                for variant_id in variant_batch:
+                    # First we store the scores for each variant in a dictionary
+                    variant = variant_batch[variant_id]
+                    rank_score_entry = variant['info_dict'].get(f'{rank_score_type}', '')
+
+                    # We need to loop through the families
+                    # This entry looks like <family_id>:<rank_score>, <family_id>:<rank_score>
+                    for family_rank_score in rank_score_entry.split(','):
+                        family_rank_score = family_rank_score.split(':')
+
+                        #TODO check if correct family id
+                        # Right now we assume that there is only one family in the vcf
+                        family_id = family_rank_score[0]
+                        rank_score = float(family_rank_score[-1])
+
+                        rank_scores[rank_score_type][variant_id] = rank_score
```

Comment:
```suggestion

                 rank_scores[rank_score_type][variant_id] = float(family_rank_score[-1])
```
I think the indentation in this comment will be wrong, but if you do not use rank_score later - skip it